common/const/strengths.ts

```diff
@@ -11,4 +11,5 @@ export const STRENGTHS: Record<HermitTypeT, Array<HermitTypeT>> = {
 	redstone: ['pvp'],
 	speedrunner: ['miner', 'prankster'],
 	terraform: ['redstone'],
+	coin: [],
```

Comment:
I know there are no hermits with this item type, but a) if a new item type is being added there should be and b) it's bring to have the same weaknesses as balanced, maybe it should be weak against itself?

Reply:
I don't think it actually needs to be here, as it isn't really a type, just energy for any type cards